### PR TITLE
12.0 mig auth oauth multi token

### PR DESCRIPTION
**Pull request by sla-subteno-it**

### CODE REVIEW

**Review by gurneyalex — APPROVED** [review]
code review, no test

File: auth_oauth_multi_token/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+# Copyright 2017 Camptocamp
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+
+{
+    'name': 'OAuth Multi Token',
+    'version': '12.0.1.0.0',
+    'license': 'AGPL-3',
+    'author': 'Florent de Labarre, '
+              'Camptocamp, '
+              'Odoo Community Association (OCA)',
```
Comment (gurneyalex) [inline auth_oauth_multi_token/__manifest__.py:11]:
```suggestion
              'Odoo Community Association (OCA)'
```

**Review by gurneyalex — CHANGES_REQUESTED** [review]

File: auth_oauth_multi_token/__manifest__.py (hunk 1 above)
Comment (pedrobaeza) [reply auth_oauth_multi_token/__manifest__.py:11]: That comma is needed for proper dictionary format (it's outside the quote).

Comment (pedrobaeza) [reply auth_oauth_multi_token/__manifest__.py:11]: Sorry, I see now that they have added the new author after this...

But I don't consider this kind of migration should rise them to co-authors, so the solution is to remove that author.

**Comment by fmdl** [comment]
move here https://github.com/OCA/server-auth/pull/106

**Comment by emagdalenaC2i** [comment]
@fmdl and @sla-subteno-it should this PR be closed and superseed in #106 ?

**Comment by fmdl** [comment]
@emagdalenaC2i this PR doesn't work. Yes is superceded by https://github.com/OCA/server-auth/pull/106

**Comment by emagdalenaC2i** [comment]
@fmdl So this PR should be closed

**Pull request closed without merging**